### DISCUSSION TITLE
Setting to raise log commit limit

### DISCUSSION THREAD

**Discussion by sdhoward** [magit/magit#4691] [Support] [answered]
When I press `l` to log recent commits in the [log buffer](https://magit.vc/manual/magit/Log-Buffer.html), I wish to see more than 256 entries in the log. I know I can press `+` to double the amount shown, or change the `-n` option every time I open the log. But I want this to be done for me every time.

If I run `magit-log-double-commit-limit`, the setting only applies to the current log window.

Is there a global setting for this? I also want to disable the graph by default.

**Comment by tarsius** [comment, marked as answer]
The infix arguments used by a certain transient popups can be [set/saved](https://magit.vc/manual/transient/Saving-Values.html) for future invocations.  TL;DR Enter the transient as usual, enable and/or disable arguments, and finally type <kbd>C-x s</kbd> or <kbd>C-x C-s</kbd>.

<sub>This is a [canned response](https://help.github.com/en/github/writing-on-github/working-with-saved-replies); it may not apply 100% in this case.</sub>
